awc/src/builder.rs

```diff
@@ -1,5 +1,7 @@
 use std::{convert::TryFrom, fmt, net::IpAddr, rc::Rc, time::Duration};
 
+use base64::{prelude::BASE64_STANDARD, Engine};
```

Comment:
if prelude is being used, then lets just use it

```suggestion
use base64::prelude::*;
```